components/toast/theme.css

```diff
@@ -57,13 +71,36 @@
   margin-right: var(--toast-spacing);
 }
 
-.is-exiting,
-.is-entering {
+.enter {
   opacity: 0;
   transform: translateY(100%);
+  max-height: 0;
 }
-
-.is-entered {
+.enter-active {
   opacity: 1;
   transform: translateY(0%);
+  max-height: 180px;
+  transition: opacity 0.3s var(--toast-animation-curve), transform 0.3s var(--toast-animation-curve),
+    max-height 0.3s var(--toast-animation-curve);
+}
+.enter-done {
+  opacity: 1;
+}
+.exit {
+  opacity: 1;
+  max-height: 180px;
+  transform: translateX(0);
```

Comment:
Could you use translate3D? https://jsperf.com/translate3d-vs-xy

Reply:
Thanks for the tip, did not really know about the performance difference

 :ok: by @Kemosabert on 29/06/2018 13:17:04